pages/content/reports/2022/index.md

```diff
@@ -0,0 +1,391 @@
+---
+author: "Open Web Docs staff"
+title: "Open Web Docs Impact and Transparency Report 2022"
+date: "2023-02-22"
+ShowToc: true
+ShowBreadCrumbs: false
+---
+
+## Executive summary
+
+Open Web Docs (OWD) is an open source collective that employs technical writers to publicly document open web technologies. OWD currently employs 4 full time writers and one part time writer, all of whom have extensive experience with Web standards documentation and MDN Web Docs. The OWD team writes new documentation, updates existing documentation, and improves documentation infrastructure.
+
+Open Web Docs’ main focus has been contributions to [MDN Web Docs](https://developer.mozilla.org), the premier platform for Web platform documentation.
+
+In 2022, OWD writers were the main contributors to MDN Web Docs.
```

Comment:
```suggestion
In 2022, OWD writers made major contributions to MDN Web Docs:
```

Reply:
maybe "a key contributor" or something a little stronger?

Reply:
"main" is correct here and we can be proud of it.